src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/tasks/CleanupEnvelopesDlqTask.java

```diff
@@ -113,10 +113,11 @@ private void logMessage(IMessage msg) {
 
     private boolean canBeCompleted(IMessage message) {
         Instant cutoff = Instant.now().minus(this.ttl);
-        Map<String, String> messageProperties = message.getProperties();
+        Map<String, Object> messageProperties = message.getProperties();
 
-        if (!CollectionUtils.isEmpty(messageProperties) && !isNullOrEmpty(messageProperties.get("deadLetteredAt"))) {
-            Instant deadLetteredAt = Instant.parse(messageProperties.get("deadLetteredAt"));
+        if (!CollectionUtils.isEmpty(messageProperties)
+            && !isNullOrEmpty((String) messageProperties.get("deadLetteredAt"))) {
```

Comment:
`(String) messageProperties.get("deadLetteredAt")` can be extracted become better readable :)

Reply:
Not from this PR, but maybe we can get rid of this condition: `CollectionUtils.isEmpty(messageProperties)`?

Reply:
we do not need string anymore and i had updated.